specs/waku/waku-1.md

```diff
@@ -303,7 +302,15 @@ Each node SHOULD broadcast its rate limits to its peers using the rate limits pa
 
 Each node SHOULD respect rate limits advertised by its peers. The number of packets SHOULD be throttled in order not to exceed peer's rate limits. If the limit gets exceeded, the connection MAY be dropped by the peer.
 
-##### Message Confirmations Field
+##### light Node Field
+
+Light Node represents a boolean option. When true, the peer SHOULD NOT forward Envelopes through the Messages packet (`0x01`).
```

Comment:
This is to inform a peer that this node is a light node, and thus that it will not relay any messages (and as such the peer might decide to disconnect). 
The current sentence sounds ambiguous to me and could be interpreted as if the peer should not forward envelopes to the peer with the light node setting.

I'm also not sure if we should repeat the boolean part here. Redundant because of ABNF spec

Reply:
>The current sentence sounds ambiguous to me and could be interpreted as if the peer should not forward envelopes to the peer with the light node setting.

That's true, let me amend this.

>I'm also not sure if we should repeat the boolean part here. Redundant because of ABNF spec

The ABNF spec defines `light-node` as a `BIT` which is implicitly a boolean, but I wanted to make it explicitly clear.

I think ideally this field should have been named `light-node-enabled` (like `confirmations-enabled`) making it clear in the name that this field has a binary state. Perhaps we could add this to the list of changes for Waku V2?

Reply:
Actually on reflection, I'll remove the mention of boolean, as the next sentence makes it clear that it is a boolean with "When true ..."